packages/components/stories/icon/icons.js

```diff
@@ -57,7 +57,12 @@ export const icons =
         'IcAppstoreWalletUsdcLight',
         'IcAppstoreWalletUsdtLight',
         'IcAppstoreWalletsLink',
-        'IcAppstoreWarning',
+        'IcAppstoreWalletsUpgradeCoinsHorizontal',
```

Comment:
can we remove this icons ? @maryia-matskevich-deriv

Reply:
done

Reply:
done